test/mock_firebase.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+
+/// Source : https://stackoverflow.com/questions/63662031/how-to-mock-the-firebaseapp-in-flutter
```

Comment:
great to put comment of reference link, but can you remove the copyright? or change it to my name :)